.git-hooks/pre-commit

```diff
@@ -1,78 +0,0 @@
-#!/usr/bin/env ruby
```

Comment:
I replaced this with a simpler bash script that is run as part of the `pre-commit-config` instead of requiring a separate git installation using `make install-git-hooks`.